system-tests/README.md

```diff
@@ -4,10 +4,10 @@ These tests deploy the application in a docker environment, and use the generate
 
 ### Running test locally
 
-Build the application launcher:
+Build the application launchers:
 
 ```
-./gradlew :launcher:shadowJar
+./gradlew shadowJar
```

Comment:
I get `registration-service    | org.eclipse.dataspaceconnector.core.security.azure.AzureVaultException: Either 'edc.vault.certificate' or 'edc.vault.clientsecret' must be supplied but both were null` when running integration tests locally

Reply:
Do you need `./gradlew -DuseFsVault="true" :launcher:shadowJar :system-tests:launchers:identity-hub:shadowJar` here?